Failure with losetup on first attempt to create a ISO image

When attempting to use this example, sometimes the ISO creation will fail on Fedora 39 Kinoite System:
```
docker run --rm --privileged --volume .:/isogenerator/output -e VERSION=38 -e IMAGE_NAME=bluefin -e IMAGE_TAG=gts -e VARIANT=Silverblue ghcr.io/ublue-os/isogenerator:38
```

Error received:

```
Traceback (most recent call last):
  File "/usr/sbin/lorax", line 223, in <module>
    main()
  File "/usr/sbin/lorax", line 204, in main
    lorax.run(dnfbase, opts.product, opts.version, opts.release,
  File "/usr/lib/python3.12/site-packages/pylorax/__init__.py", line 324, in run
    rc = rb.create_ext4_runtime(joinpaths(installroot,runtime),
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pylorax/treebuilder.py", line 254, in create_ext4_runtime
    imgutils.mkrootfsimg(self.vars.root, joinpaths(workdir, "LiveOS/rootfs.img"),
  File "/usr/lib/python3.12/site-packages/pylorax/imgutils.py", line 136, in mkrootfsimg
    mkext4img(rootdir, outfile, label=label, size=fssize)
  File "/usr/lib/python3.12/site-packages/pylorax/imgutils.py", line 586, in mkext4img
    mkfsimage("ext4", rootdir, outfile, size, mountargs=mountargs,
  File "/usr/lib/python3.12/site-packages/pylorax/imgutils.py", line 555, in mkfsimage
    with LoopDev(outfile, size) as loopdev:
  File "/usr/lib/python3.12/site-packages/pylorax/imgutils.py", line 357, in __enter__
    self.loopdev = loop_attach(self.filename)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pylorax/imgutils.py", line 204, in loop_attach
    dev = runcmd_output(["losetup", "--find", "--show", outfile]).strip()
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pylorax/executils.py", line 373, in runcmd_output
    return execWithCapture(cmd[0], cmd[1:], **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pylorax/executils.py", line 251, in execWithCapture
    return _run_program(argv, stdin=stdin, root=root, log_output=log_output, filter_stderr=filter_stderr,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/pylorax/executils.py", line 205, in _run_program
    raise subprocess.CalledProcessError(proc.returncode, argv, output)
subprocess.CalledProcessError: Command '['losetup', '--find', '--show', '/var/tmp/lorax/lorax.8inbedip/installroot/images/runtime-workdir/LiveOS/rootfs.img']' returned non-zero exit status 1.
2024-02-21 21:24:31,337: Cleaning up tempdir - /var/tmp/lorax/lorax.8inbedip
make: *** [Makefile:51: boot.iso] Error 1
```

--- a/Containerfile
+++ b/Containerfile
@@ -21,4 +21,4 @@ RUN dnf install -y make && make install-deps
 
 VOLUME /isogenerator/output
 
-ENTRYPOINT ["sh", "-c", "make output/${IMAGE_NAME}-${IMAGE_TAG}.iso ARCH=${ARCH} VERSION=${VERSION} IMAGE_REPO=${IMAGE_REPO} IMAGE_NAME=${IMAGE_NAME} IMAGE_TAG=${IMAGE_TAG} VARIANT=${VARIANT} WEB_UI=${WEB_UI}"]
+ENTRYPOINT /isogenerator/entrypoint.sh

--- a/README.md
+++ b/README.md
@@ -16,14 +16,22 @@ This will create an ISO with the baked in defaults of the container image.
 See [Customizing](#customizing) for information about customizing the image that gets created. The variable can either be defined as environment variables or as command arguments.
 Examples:
 
-Creating Bluefin GTS ISO
+Creating Universal Blue Silverblue ISO
 ```bash
-docker run --rm --privileged --volume .:/isogenerator/output -e VERSION=38 -e IMAGE_NAME=bluefin -e IMAGE_TAG=gts -e VARIANT=Silverblue ghcr.io/ublue-os/isogenerator:38
+docker run --rm --privileged --volume .:/isogenerator/output -e VERSION=39 -e IMAGE_NAME=silverblue-main -e IMAGE_TAG=latest -e VARIANT=Silverblue ghcr.io/ublue-os/isogenerator:39
+```
+```bash
+# Requires sudo to run
+sudo podman run --rm --privileged --volume .:/isogenerator/output -e VERSION=39 -e IMAGE_NAME=silverblue-main -e IMAGE_TAG=latest -e VARIANT=Silverblue ghcr.io/ublue-os/isogenerator:39
 ```
 
-Creating Bazzite Latest ISO
+Creating Universal Blue Kinoite ISO
 ```bash
-docker run --rm --privileged --volume .:/isogenerator/output -e VERSION=39 -e IMAGE_NAME=bazzite -e IMAGE_TAG=latest -e VARIANT=Kinoite ghcr.io/ublue-os/isogenerator:39
+docker run --rm --privileged --volume .:/isogenerator/output -e VERSION=39 -e IMAGE_NAME=kinoite-main -e IMAGE_TAG=latest -e VARIANT=Kinoite ghcr.io/ublue-os/isogenerator:39
+```
+```bash
+# Requires sudo to run
+sudo podman run --rm --privileged --volume .:/isogenerator/output -e VERSION=39 -e IMAGE_NAME=kinoite-main -e IMAGE_TAG=latest -e VARIANT=Kinoite ghcr.io/ublue-os/isogenerator:39
 ```
 
 ## Customizing

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -1,0 +1,8 @@
+#!/bin/bash
+
+set -eu
+
+# pre-create loop devices manually. In containers we can't use losetup for that.
+mknod -m 0660 /dev/loop0 b 7 0  2> /dev/null || true
+
+make output/${IMAGE_NAME}-${IMAGE_TAG}.iso ARCH=${ARCH} VERSION=${VERSION} IMAGE_REPO=${IMAGE_REPO} IMAGE_NAME=${IMAGE_NAME} IMAGE_TAG=${IMAGE_TAG} VARIANT=${VARIANT} WEB_UI=${WEB_UI}

